resources/assets/uad_lists.json

```
@@ -4277,7 +4277,7 @@
   {
     "id": "com.samsung.android.spayfw",
     "list": "Oem",
-    "description": "Samsung Pay Framework needed for Samsung Pay\nSee below\n",
+    "description": "Samsung Pay Framework needed for Samsung Pay\nSamsung Pay is a mobile payment and digital wallet service by Samsung Electronics that lets users make payments using compatible phones \nand other Samsung-produced devices\nSee below\n",
```

```suggestion
    "description": "Samsung Pay Framework needed for Samsung Pay\nSamsung Pay is a mobile payment and digital wallet service by Samsung Electronics that lets users make payments using compatible phones and other Samsung-produced devices. See below.",
```

You don't need to end with an extra line, also removed some redundant ones.